scripts/lochness_create_template.py

```diff
@@ -203,11 +203,11 @@ def create_keyring_template(keyring_loc: Path, args: object) -> None:
         if args.enter_passwords:
             client_id = getpass.getpass('BOX CLIENT ID: ')
             client_secret = getpass.getpass('BOX CLIENT SECRET: ')
-            api_token = getpass.getpass('BOX API TOKEN: ')
+            user_id = getpass.getpass('BOX USER ID: ')
```

Comment:
I understand that the `user_id` is the only additional thing that is being asked for from the user.

Reply:
So we are no longer asking the user to provide `API TOKEN`, right?

Reply:
Correct.